src/elements/content-uploader/UploadsManagerAction.js.flow

```diff
@@ -28,11 +33,11 @@ const UploadsManagerAction = ({ onClick, hasMultipleFailedUploads }: Props) => {
 
     return (
         <div className="bcu-uploads-manager-action">
-            <PrimaryButton onClick={handleResumeClick} type="button" {...resin}>
-                <FormattedMessage {...resumeMessage} />
-            </PrimaryButton>
+            <Button onClick={handleResumeClick} size="small" variant="primary" {...resin}>
+                {intl.formatMessage(resumeMessage)}
+            </Button>
         </div>
     );
 };
 
-export default UploadsManagerAction;
+export default injectIntl(UploadsManagerAction);
```

Comment:
I need to use `intl.formatMessage()` because of [this](https://github.com/box/box-ui-elements/pull/3604#discussion_r1707959269). Is there any difference or benefits that we use `useIntl` instead?